librosa/core/constantq.py

```diff
@@ -172,46 +176,32 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
 
     # Determine required resampling quality
     Q = float(filter_scale) / (2.0**(1. / bins_per_octave) - 1)
-
     filter_cutoff = fmax_t * (1 + filters.window_bandwidth('hann') / Q)
-
     nyquist = sr / 2.0
-
     if filter_cutoff < audio.BW_FASTEST * nyquist:
         res_type = 'kaiser_fast'
-    elif filter_cutoff < audio.BW_BEST * nyquist:
-        res_type = 'kaiser_best'
     else:
         res_type = 'kaiser_best'
 
-    cqt_resp = []
-
-    len_orig = len(y)
-
     y, sr, hop_length = __early_downsample(y, sr, hop_length,
```

Comment:
I agree it's orthogonal to this PR.  I thought maybe the call to
early_resample had become more meaningless under 'kaiser_fast', but I
see it was always that way.  I'll file an issue ;-)

  DAn.

On Sun, May 8, 2016 at 5:24 PM, Brian McFee <notifications@github.com> wrote:
> In librosa/core/constantq.py:
>
>>      else:
>>          res_type = 'kaiser_best'
>>
>> -    cqt_resp = []
>> -
>> -    len_orig = len(y)
>> -
>>      y, sr, hop_length = __early_downsample(y, sr, hop_length,
>
> I agree that it's a little contorted, but I think that it's an orthogonal
> issue to those addressed in this PR. I'd vote for punting a refactor of this
> to a later date.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub